web/portal/platform/src/entities/ActiveCalls/ActiveCallsProperties.tsx

```diff
@@ -5,16 +5,9 @@ import {
 } from '@irontec/ivoz-ui/services/entity/EntityService';
 
 export type ActiveCallsPropertyList<T> = {
-  Event?: T;
```

Comment:
why?

Reply:
![image](https://user-images.githubusercontent.com/114579356/221141011-9515f507-411a-46dc-ac89-9f52e7ed16aa.png)

Reply:
this is another entity actually, data come from a web socket service, not the API (https://github.com/irontec/ivozprovider/blob/bleeding/web/portal/platform/src/router/addCustomRoutes.ts) . Will handle it on it's own issue. Revert this for now